src/bridge/jsHelpers.ts

```diff
@@ -391,16 +390,17 @@ export const makeScanAccounts =
                 }
               );
               let res = derivationsCache[freshAddressPath];
-
               if (!res) {
-                res = await getAddress(transport, {
-                  currency,
-                  path: freshAddressPath,
-                  derivationMode,
-                });
+                res =
+                  currency.family === "bitcoin"
+                    ? { path: freshAddressPath }
```

Comment:
not super confortable with this branching idea (family specific in generic codebase)

it could be a parameter of the makeScanAccounts as the fact you don't need to derivate the address (as we derivate the xpub i imagine?)